scripts/run_backend_tests.py

```diff
@@ -92,13 +99,18 @@
 TEST_RUNNER_PATH = os.path.join(os.getcwd(), 'core', 'tests', 'gae_suite.py')
 # This should be the same as core.test_utils.LOG_LINE_PREFIX.
 LOG_LINE_PREFIX = 'LOG_INFO_TEST: '
+# This path points to a JSON file that defines which modules belong to
+# each shard.
+SHARDS_SPEC_PATH = os.path.join(
```

Comment:
SPECS_SHARDS_PATH? (not sure!) [The current one sounds like a path to the shard's specs]

Reply:
I think this name is correct since this is the path to the file that specifies the shards (the shard specification)